docs/server/node/_options.mdx

```diff
@@ -19,3 +19,7 @@
   - disables all network access, so the SDK will only return default (or overridden) values.  Useful in testing
 - **initTimeoutMs**: number, default 0
   - sets a maximum time to wait for the config download network request to resolve before considering the SDK initialized and resolving the call to `initialize()`
+- **dataAdapter**: IDataAdapter, default null
+  - Replaces the in-memory data caching with an adapter that defines custom storage of config specs. Can also be used to bootstrap Statsig server.
```

Comment:
This doesnt replace the in memory storage - everything is still stored in memory. It is primarily for bootstrapping the server from your storage